tuist/tuist
# Merge 'fourier format swift' into 'fourier lint tuist'

### Short description 📝
I'm merging `fourier format swift` into `fourier lint tuist` with the goal of having a single formatting and styling command.

### How to test the changes locally 🧐

1. Run `fourier format tuist`
2. Run `fourier format tuist --fix`
3. Both commands should work successfully.

### Checklist ✅
